src/terraform.h

```diff
@@ -12,6 +12,8 @@
 
 #include <map>
 
+static const Money TERRAFORM_UNIT_COST(40);  ///< The cost for applying one elemental unit of terraforming modifications.
```

Comment:
At some point in the future you'll want to customize these, one idea is to collect them now all in one file or even ask for them using a function.
It's not much additional work now hopefully, and it makes customization simpler.